introduction.md

```diff
@@ -0,0 +1 @@
+My name's Max Wolffe. I'm a fourth year studying Electrical Engineering and Computer Science at UC Berkeley!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.